lib/push.js

```diff
@@ -178,29 +178,30 @@ module.exports = function (log, db, config) {
    * if they receive an unsupported message type.  Filter out
    * devices that we know won't respond well to the given command.
    *
-   * @param {String} command
-   * The command from the push message payload
+   * @param {Object} data
+   * The push message payload
    * @param {Device[]} devices
    * The list of devices to which to send the push.
    */
-  function filterSupportedDevices(command, devices) {
-    let requiredIOSVersion
+  function filterSupportedDevices(data, devices) {
+    const command = data && data.command
+    let canSendToDevice/* ({Number} version) => bool */
```

Comment:
naming nit: can we call this "canSendToIOSVersion" or similar?  I only ask because we had a little bit of bikeshedding in the previous version of this patch, to insist on using "IOS" in the name of the variable in the interests of clarity :-)